mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -356,7 +356,7 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
 	METHOD method_31484 streamIntoPassengers ()Ljava/util/stream/Stream;
 	METHOD method_32312 getFrozenTicks ()I
 	METHOD method_32313 getFreezingScale ()F
-	METHOD method_32314 isFreezing ()Z
+	METHOD method_32314 isFrozen ()Z
```

Comment:
This method indicates whether the entity has been in powder snow for long enough to start taking damage. The name `isFreezing` would make more sense for the currently unmapped `method_40071`, which indicates whether the entity is in powder snow and is able to freeze.